_data/cards/pt_BR/viral-marketing.yaml

```diff
@@ -0,0 +1,22 @@
+name: Marketing Viral
+logo:
+short-description:
+key-objectives:
+  - Estudar as métricas de marketing viral, aprendendo a identificar possíveis problemas e otimizar continuamente as campanhas para alcançar os melhores resultados em termos de engajamento e conversão.
+  - Compreender como o Buzz Marketing e outras estratégias virais podem ser utilizadas para ampliar significativamente o alcance de uma marca, gerando um impacto duradouro e positivo.
+  - Explorar exemplos reais de campanhas virais bem-sucedidas, analisando os elementos que contribuíram para seu sucesso e como esses fatores podem ser replicados em diferentes contextos.
+  - Identificar e trabalhar com influenciadores-chave e comunidades online para criar uma base sólida de defensores da marca, maximizando a propagação das mensagens de marketing de forma orgânica.
+contents:
+  - type: ARTICLE
+    title: "Marketing Viral: o que é, quais os benefícios da estratégia e como viralizar sua campanha?"
+    link: https://pagar.me/blog/marketing-viral/
+  - type: ARTICLE
+    title: "O que é marketing viral? Veja o conceito, como fazer e exemplos"
+    link: https://www.remessaonline.com.br/blog/marketing-viral/
+  - type: ARTICLE
+    title: "Buzz marketing: o que é, vantagens e como usar essa estratégia"
+    link:https://www.cursospm3.com.br/blog/buzz-marketing/#:~:text=Buzz%20marketing%20%C3%A9%20uma%20estrat%C3%A9gia,servi%C3%A7o%20ou%20da%20pr%C3%B3pria%20empresa.
```

Comment:
Missing space, you could also put the link inside "" to prevent some kind of bug